package.py

```diff
@@ -96,6 +96,7 @@ def __init__(self) -> None:
         os.environ.pop(CERTIFICATE_KEY, None)
         os.environ.pop(APPLE_ID, None)
         os.environ.pop(APPLE_ID_PASS, None)
+        os.environ.setdefault('CYPRESS_INSTALL_BINARY', '0')
```

Comment:
is this related to the removal of the no-optional?

Reply:
yes this stops the installation from packaging from downloading the Cypress binary when installing because we don't really need it, the whole idea behind --no-optional was to avoid downloading cypress on packaging and save some time.